src/shared/conflicts.ts

```diff
@@ -26,15 +27,14 @@ export const findConflictsInComponentSet = (cs: ComponentSet, conflicts: ChangeR
   // map do dedupe by name-type-filename
   const conflictMap = new Map<string, ConflictResponse>();
   conflicts
-    .filter((cr) => cr.name && cr.type && cs.has({ fullName: cr.name, type: cr.type }))
+    .filter(isChangeResultWithNameAndType)
```

Comment:
`isChangeResultWithNameAndType` narrows the `ConflictResponse`  so that other functions can depend on those props existing